### DISCUSSION TITLE
GUID vs Autonumeric

### DISCUSSION THREAD

**Discussion by lucasrochadejesus** [balta-io/forum#87] [Tira Dúvidas] [answered]
Fala Balta tranquilo? cara ate então trabalhei com MySQL e sempre utilizei Auto Numeric pra gerar. Ha alguns meses comecei a trabalhar numa empresa e-commerce com SQL Server e eles nao utilizam 1,2,3... usam somente GUID.

Tem algum curso que voce aborda sobre isso? lembro de ter dado uma assitida rapida na parte de SQL Server e voce falando que iria abordar isso em outro momento..

Abs.

**Comment by andrebaltieri** [comment, marked as answer]
@lucasrochadejesus temos alguns sim! Eles estão na carreira .NET
https://balta.io/carreiras/desenvolvedor-backend-dotnet

É interessante fazer os cursos de SQL e Dapper também, para entender os prós e contras de usar GUID ou INT.
